src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt

```diff
@@ -142,14 +139,16 @@ class HazelcastOrganizationService(
             else -> throw IllegalStateException("Only users and roles can create organizations.")
         }//Fall through by design
 
-        initializeOrganizationPrincipals(principal, organization)
+        initializeOrganizationPrincipal(principal, organization)
         initializeOrganization(organization)
         organizationMetadataEntitySetsService.initializeOrganizationMetadataEntitySets(organization.id)
```

Comment:
Why not move this to the very end?

Reply:
I was trying to avoid reordering things that were working, but this seems like it should be safe to move. What would be the reason to move it?